coul.c

```diff
@@ -2916,7 +2918,13 @@ int main(int argc, char **argv, char **envp) {
 
     init_post();
     report_init(stdout, argv[0]);
+    if (freopen(rpath, "a+", rfp) == NULL)
+       fail("could not reopen %s: %s", rpath, strerror(errno));
+    setlinebuf(rfp);
     if (rfp) report_init(rfp, argv[0]);
+    if (freopen(rpath, "a+", rfp) == NULL)
+       fail("could not reopen %s: %s", rpath, strerror(errno));
+    setlinebuf(rfp);
```

Comment:
It makes no sense to me that this would be needed again here just because report_init() wrote a line.

Reply:
But we need to redefine (for Windows) after this function the file stream . But here we need to clarify this point with programmers who write programs for Windows.

Reply:
I don't understand, sorry. We have just called a function that will write a line to the output stream, this is a perfectly normal thing to do. Please try to explain in more detail why you think the file should be reopened after that (and not, for example, after every other time we write a line of logging to `rfp`).